src/librustc/metadata/loader.rs

```diff
@@ -248,6 +249,7 @@ pub struct Context<'a> {
     pub ident: &'a str,
     pub crate_name: &'a str,
     pub hash: Option<&'a Svh>,
+    pub target: &'a Target,
```

Comment:
The `creader.rs` changes sets it to either &self.sess.target.target or &self.sess.host depending on whether it looks for target or host libraries.